docs/remix_commands.md

```diff
@@ -28,66 +25,22 @@ In the console, you can run the commands listed below.  Once you start to type a
 
 **remix.loadurl(url)**: Load the given url in the file explorer. The url can be of type github, swarm or ipfs.
 
-**remix.setFile(path, content)**: set the content of the file located at the given path
```

Comment:
Is this command not working for remix anymore?

Reply:
input: remix.setFile("./contracts/f", "fred")
returns: remix.setFile is not a function

Reply:
ok